docs/FunctionSamples/README.md

```diff
@@ -0,0 +1,19 @@
+# Microsoft Azure Functions in Java
+
+This repo contains Java Azure Functions annotation samples for different events.
+
+## Prerequisites
+- JDK 1.8
+- Maven 3.0+
+- [.NET Core SDK](https://www.microsoft.com/net/learn/get-started/windows)
+- [Azure Functions Core Tools v2](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local#v2)
+- [Maven Plugin for Azure Functions](https://github.com/Microsoft/azure-maven-plugins/blob/develop/azure-functions-maven-plugin/README.md)
```

Comment:
We should use relative path here, then it will point to the README file in different branches